README.md

```diff
@@ -29,11 +29,12 @@ It moved to the [Akka](https://github.com/akka/) organisation in 2016 and the fi
 
 ## Branches and versions
 
-There are three active branches of development:
+There are three branches of development:
 
+* 1.0 -> `master` - current active development and stable 1.0.x patch releases
+* 0.80+ (currently 0.100) -> `release-0.x`  - removed use of Cassandra Materialized Views after they were marked as not to be used in production. 
 * 0.50+ (currently 0.62) -> `release-0.50`- first release under this organisation, previously under krasserm. No planned releases for this version.
-* 0.80+ (currently 0.100) -> `release-0.x`  - removed use of Cassandra Materialized Views after they were marked as not to be used in production. Current stable version.
-* 1.0 -> `master` - not yet released, planned upgrade to 4.0.x of the Cassandra Driver that includes breaking API changes to CassandraSession
 
-To release a 0.50+ or 0.80+ version checkout that branch and see the release instructions there. 
+## License
```

Comment:
Add a note about support from Lightbend.

Reply:
we have https://github.com/akka/akka-persistence-cassandra/issues/751 for that